app/components/trainees/confirmation/degrees/view.rb

```diff
@@ -2,10 +2,11 @@ module Trainees
   module Confirmation
     module Degrees
       class View < GovukComponent::Base
-        attr_accessor :degrees
+        attr_accessor :degrees, :trainee
 
         def initialize(degrees:)
           @degrees = degrees
+          @trainee = degrees.first.trainee
```

Comment:
Don't think we need this we can get it inside the `get_degree_rows` method.

Reply:
https://github.com/DFE-Digital/register-trainee-teachers/blob/00e528eca943f000ff738a12000017ceca42da48/app/components/trainees/confirmation/degrees/view.html.erb#L14

Reply:
# 🤦

Reply:
# 🤦